### ISSUE TITLE
Pack selection keeps resetting

### ISSUE THREAD

**Issue by BigDub** [jakelauer/allbadcards#28]
1. What URL was your game at?
allbad.cards/game/

2. Describe the problem you experienced
- Sitting in the lobby for a while waiting for people to join, I noticed that the pack selection keeps getting reset to default.

3. Did you see any errors?
- I go through and select my packs. Then I scroll back up and wait in the lobby. After a while the pack selection resets. It goes back down from the 26 I selected back to the default 20.

4. Screenshots - If applicable, add screenshots to help explain your problem.

5. Additional information
Please provide any additional helpful information, like:
- This occurs, and reverts, both when I have added packs or removed packs from the default selection.

**Comment by jakelauer** [comment]
Were you refreshing the page? The selection will not stay if you keep refreshing, or if you go to the page in another tab or something.

**Comment by BigDub** [comment]
I was not. I kept open only the single tab in my chrome browser and watched
it.
I believe it happened in both incognito and the normal browser.
If I had to guess, I would say something was timing out and resetting to
default somehow.
I would select the packs I wanted, and then provide no further input and
just watch the screen. After a minute or so, the pack selection went back
to default every time I tried.
